cookiecutter-django-app/{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -25,7 +25,7 @@ def get_version(*file_paths):
     Extract the version string from the file at the given relative path fragments.
     """
     filename = os.path.join(os.path.dirname(__file__), *file_paths)
-    version_file = open(filename).read()
+    version_file = open(filename, encoding="utf8").read()
```

Comment:
pylint version [2.10.0](https://github.com/PyCQA/pylint/releases/tag/v2.10.0) deprecated the use of open() without explicitly specifying any encoding. See https://github.com/PyCQA/pylint/issues/3826 for details.